CMR/python/test/cmr/auth/test.py

```diff
@@ -0,0 +1,129 @@
+"""
+NASA EO-Metadata-Tools Python interface for the Common Metadata Repository (CMR)
+
+    https://github.com/nasa/Common-Metadata-Repository/
+
+Copyright (c) 2020 United States Government as represented by the Administrator
+of the National Aeronautics and Space Administration. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+"""
+#Test cases for the cmr.auth package
+#date 2020-10-15
+#since 0.0
+
+from unittest.mock import Mock
+from unittest.mock import patch
+import json
+import unittest
+import urllib.error as e
```

Comment:
I'm not a huge fan of 1 letter aliases

Reply:
your probably unhappy with a lot in CMR then 👍

Reply:
Largely irrelevant, just because someone did it wrong somewhere else doesn't mean we should do it wrong here.  We also don't code in python so I don't know what the established coding style is, but for clojure we use full names except in specific circumstances.
https://stuartsierra.com/2015/05/10/clojure-namespace-aliases

Reply:
I found a couple more and got rid of them